unicorn.js

```diff
@@ -0,0 +1,192 @@
+    'unicorn/expiring-todo-comments': 'error',
+
+    // Enforce explicitly comparing the length property of a value
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/explicit-length-check.md
+    'unicorn/explicit-length-check': 'error',
+
+    // Enforce importing index files with .
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/import-index.md
+    'unicorn/import-index': 'error',
+
+    // Enforce the use of new for all builtins, except String, Number and Boolean
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/new-for-builtins.md
+    'unicorn/new-for-builtins': 'error',
+
+    // Enforce specifying rules to disable in eslint-disable comments
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-abusive-eslint-disable.md
+    'unicorn/no-abusive-eslint-disable': 'error',
+
+    // Require Array.isArray() instead of instanceof Array
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-array-instanceof.md
+    'unicorn/no-array-instanceof': 'error',
+
+    // Do not use leading/trailing space between console.log parameters
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-console-spaces.md
+    'unicorn/no-console-spaces': 'error',
+
+    // Do not use a for loop that can be replaced with a for-of loop
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-for-loop.md
+    'unicorn/no-for-loop': 'error',
+
+    // Enforce the use of Unicode escapes instead of hexadecimal escapes
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-hex-escape.md
+    'unicorn/no-hex-escape': 'error',
+
+    // Disallow identifiers starting with new or class
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-keyword-prefix.md
+    'unicorn/no-keyword-prefix': 'off',
+
+    // Disallow nested ternary expressions
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-nested-ternary.md
+    'unicorn/no-nested-ternary': 'error',
+    // Improved by 'unicorn/no-nested-ternary'
+    // https://eslint.org/docs/rules/no-nested-ternary
+    'no-nested-ternary': 'off',
+
+    // Enforce the use of Buffer.from() and Buffer.alloc() instead of the deprecated new Buffer().
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-new-buffer.md
+    'unicorn/no-new-buffer': 'error',
+
+    // Disallow process.exit()
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-process-exit.md
+    'unicorn/no-process-exit': 'error',
+
+    // Disallow unreadable array destructuring
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-unreadable-array-destructuring.md
+    'unicorn/no-unreadable-array-destructuring': 'error',
```

Comment:
This would restrict certain things like:
```
const [,, third] = someArray;
```

Reply:
yeah, i think thats the intention but i don't necessarily agree. happy to remove.